composables/__tests__/useUserOrder.test.ts

```diff
@@ -0,0 +1,67 @@
+import { useUserOrder } from "../useUserOrder"
+
+const searchOrderResponse = {
+  data: {
+    orders: {
+      pageCount: 0,
+      items: [
+        {
+          submittedDate: null,
+          id: "1313b457213c7a00016b1923000045a4",
+          orderNumber: 36,
+          status: "Pending",
+          total: 3425,
+          items: [
+            {
+              product: {
+                productCode: "MS-JKT-013",
+                name: "Stingray Shell Jacket",
+              },
+              quantity: 5,
+              total: 1000,
+            },
+            {
+              product: {
+                productCode: "MS-JKT-004",
+                name: "Condor Triclimate 3-in-1 Jacket",
+              },
+              quantity: 2,
+              total: 400,
+            },
+            {
+              product: {
+                productCode: "MS-CAM-004",
+                name: "GoPro Hero3 Mount",
+              },
+              quantity: 1,
+              total: 125,
+            },
+          ],
+        },
+      ],
+    },
+  },
```

Comment:
As we are mocking response, even if there will be a single object inside items[] it will be fine

Reply:
Resolved